backend/sites/src/app/resolvers/dropdown/dropdown.resolver.ts

```diff
@@ -204,4 +204,45 @@ export class DropdownResolver {
       );
     }
   }
+
+  @Roles({
+    roles: [    
+      CustomRoles.Internal,
+      CustomRoles.SiteRegistrar,
+    ],
+    mode: RoleMatchingMode.ANY,
+  })
+  @Query(()=>DropdownResponse,{name: 'getIDIRUserListForDropDown'})
+  async getIDIRUserListForDropDown() {
+    try {
+      const result =
+        await this.dropdownService.getIDIRUserGivenNamesForDropDown();
+      if (result.length > 0) {
```

Comment:
can result be null or empty ?

Reply:
no, incase of no results empty array is returned to controller